docs/architecture/microservices/net-core-net-framework-containers/container-framework-choice-factors.md

```diff
@@ -11,17 +11,17 @@ The following decision table summarizes whether to use .NET Framework or .NET Co
 > Your development machines will run one Docker host, either Linux or Windows. Related microservices that you want to run and test together in one solution will all need to run on the same container platform.
 
 | Architecture / App Type | Linux containers | Windows Containers |
-|-------------------------|------------------|--------------------|
+|-------------------------|:----------------:|:------------------:|
 | Microservices on containers | .NET Core | .NET Core |
```

Comment:
I actually prefer the current alignment on this table.
This is what it will look like when merged:
![image](https://user-images.githubusercontent.com/12971179/74875097-96a4f380-5316-11ea-88d8-77626e0d2377.png)

vs.
https://docs.microsoft.com/en-us/dotnet/architecture/microservices/net-core-net-framework-containers/container-framework-choice-factors

Reply:
Well, now that you mention it...

The current one is easier to read, I'll revert.

I guess you'll have to approve again, so it'll only be the `all` to `most` change 😁